test/test_real_pebble.py

```diff
@@ -56,13 +55,17 @@ def get_socket_path() -> str:
     return socket_path
 
 
-@unittest.skipUnless(os.getenv('RUN_REAL_PEBBLE_TESTS'), 'RUN_REAL_PEBBLE_TESTS not set')
-class TestRealPebble(unittest.TestCase):
-    def setUp(self):
-        self.client = pebble.Client(socket_path=get_socket_path())
+@pytest.mark.skipif(
+    os.getenv('RUN_REAL_PEBBLE_TESTS') != '1',
+    reason='RUN_REAL_PEBBLE_TESTS not set',
+)
+class TestRealPebble:
+    @pytest.fixture
+    def client(self):
+        return pebble.Client(socket_path=get_socket_path())
```

Comment:
I'd move this fixture to the module level.

As far as I can see the 3 copies in this file are all the same.

Same for others files, thugh I didn't check those carefully.

Reply:
I will fix this and test it in real pebble test.